autoplay/data_tools/last_fm_data.py

```diff
@@ -49,7 +93,13 @@ def normalize_scrobble(scrobble: NamedTuple):
         [type]: [description]
     """
     parsed_date = datetime.strptime(scrobble.playback_date, LAST_FM_TIMESTAMP_FORMAT)
-    normalized = [str(scrobble.track), str(scrobble.album), parsed_date]
+    #top_tags = get_top_tags(scrobble)
+    top_tags=[]
+    try:
+        artist, title = str(scrobble.track).split(" - ", 1)
+    except ValueError:
+        print(str(scrobble.track))
+    normalized = [title, str(scrobble.album), artist, top_tags, parsed_date]
```

Comment:
This seems brittle, as in some songs will have a dash in them or artists...should be a little more robust